src/compiler/binder.rs

```diff
@@ -382,6 +385,13 @@ impl BinderType {
                 symbol_flags,
                 symbol_excludes,
             )),
+            SyntaxKind::TypeAliasDeclaration => Some(self.declare_symbol(
```

Comment:
This isn't doing anything yet but I think this is where type parameters on generic type alias declarations will get put into its `locals` symbol table